renpy/display/behavior.py

```diff
@@ -1130,6 +1130,13 @@ def input_post_per_interact():
             i.caret_pos = len(content)
 
 
+def blink(trans, st, at, caret_blink):
+    if int(st*2/caret_blink)%2:
+        trans.alpha = .0
+    else:
+        trans.alpha = 1.
+    return (caret_blink/2)-(st%(caret_blink/2))
```

Comment:
I think we can simplify this down too, to use fewer ops and even managing to do away with the conditional:
```suggestion
    def blink(trans, st, at, caret_blink):
        ttl = caret_blink - st % caret_blink
        trans.alpha = ttl > caret_blink / 2
        return ttl % caret_blink / 2

```

Reply:
Shouldn't the last line be `ttl % (caret_blink/2.)` ?

Reply:
Yup, you're right /o\ I've updated it in the suggestion accordingly. 👍